src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write code here
+document.querySelectorAll('.tree > li').forEach((item) => {
+  const span = document.createElement('span');
+
+  span.textContent = item.firstChild.textContent.trim();
+
+  item.firstChild.replaceWith(span);
+
+  span.addEventListener('click', () => {
+    const subList = item.querySelector('ul');
+
+    if (subList) {
+      subList.style.display =
+        subList.style.display === 'none' ? 'block' : 'none';
```

Comment:
The line `subList.style.display` should be initialized to 'none' if it is not already set. This ensures that the toggle functionality works correctly when the page is first loaded. Consider adding a check to set `subList.style.display` to 'none' if it is not already defined.